lib/auth/permissions.go

```diff
@@ -322,6 +322,7 @@ func GetCheckerForBuiltinRole(clusterName string, clusterConfig services.Cluster
 			services.RoleSpecV3{
 				Allow: services.RoleConditions{
 					Namespaces: []string{services.Wildcard},
+					AppLabels:  services.Labels{services.Wildcard: []string{services.Wildcard}},
```

Comment:
Can you remove this line and share the test that fails? I wonder if we're dealing with an incorrect test or incorrect implementation.

Reply:
@russjones With this removed, it's TestAuthTokens and TLSSuite.TestAppServerCRUD that are failing

Reply:
Unclear why App should be able to read all apps. Probably App should be able to read it's own record instead.